packages/manhattan/src/index.js

```diff
@@ -196,21 +168,21 @@ const ManhattanPlot = ({
 ManhattanPlot.propTypes = {
   data: PropTypes.arrayOf(PropTypes.shape({
     snp: PropTypes.string.isRequired,
```

Comment:
Eventually we'll want to change snp to something that plots the more general case of a locus (chrom-pos or xpos), so that plot can support genes as well.